pkg/curatedpackages/bundle.go

```diff
@@ -2,18 +2,27 @@ package curatedpackages
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"strings"
 
-	"sigs.k8s.io/yaml"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
 
 	packagesv1 "github.com/aws/eks-anywhere-packages/api/v1alpha1"
 	"github.com/aws/eks-anywhere/pkg/constants"
 	"github.com/aws/eks-anywhere/pkg/logger"
 	releasev1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
 )
 
+var scheme = runtime.NewScheme()
```

Comment:
is this being used?

Reply:
It should be used a few lines below, in the init() method. Github only shows the one line you highlighted, but if you were to look below, it should look like this:

```
var scheme = runtime.NewScheme()

func init() {
	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
	utilruntime.Must(packagesv1.AddToScheme(scheme))
}
```

Reply:
Ahh yeah I see that. However, aside from adding those crds to the scheme, are we using the scheme for anything else? It seems like we are creating a new one, so it's not necessarily modifying a global one (global at the client level, obv this one is global for this package).

Reply:
Oh yes, it's doing nasty side-effecty-stateful things inside of the utilruntime package (hey, I didn't write it). As I'm sure you know, without it we couldn't marshal/unmarshal the packagesv1 (aka curated packages) API types. We need to do that now since we're using the go client directly, whereas before it was done via manual calls to `yaml.Unmarshal` etc.

Reply:
I see. But that's what's tripping me, how is this having side effects when we are creating our own scheme? This is mostly out of curiosity btw. But every time I have seen this been used is either usually over `scheme.Scheme` or if the new created scheme was being passed to a client. But it doesn't seem like here we are doing either.